### ISSUE TITLE
No Data is Available

### ISSUE THREAD

**Issue by sabin-ittihad** [trinity652/Document-Forgery-Detection#5]
I have setup the project and uploaded the document, but no information is being displayed in the dashboard, or anywhere.

Is it still incomplete so that we can add some functionality? Or could you please add some information about the steps to follow.

**Comment by trinity652** [comment]
Yes, it is incomplete as it was created for a hackathon, you are free to clone and add the functionality, I will add functionality if get time.
